### PR TITLE
lightning: remove default backend setting from the configuration

### PR DESCRIPTION
**Pull request by gozssky**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Remove default backend setting from the configuration and move local backend to the first. 

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@gozssky Please fix the CI lint and specify the applicable version(s). Thanks~

**Comment by glorv** [comment]
LGTM

**Comment by TomShawn** [comment]
@gozssky Is this PR content applicable only to the dev version?

**Comment by gozssky** [comment]
> @gozssky @glorv Is this PR content applicable only to the dev version?

I think this is applicable only to the dev version. What do you think? @glorv @kennytm

**Comment by kennytm** [comment]
@TomShawn yes, it is only available on dev version (and will be carried to 5.2 and 6.0)

**Review by TomShawn — APPROVED** [review]
LGTM

File: tidb-lightning/tidb-lightning-backends.md [hunk 1]
```diff
@@ -8,16 +8,16 @@ aliases: ['/docs-cn/dev/tidb-lightning/tidb-lightning-backends/','/docs-cn/dev/r
 
 TiDB Lightning 的[后端](/tidb-lightning/tidb-lightning-glossary.md#backend)决定 `tidb-lightning` 组件将如何把将数据导入到目标集群中。目前，TiDB Lightning 支持以下后端：
 
-+ [Importer-backend](#tidb-lightning-importer-backend)（默认）
 + [Local-backend](#tidb-lightning-local-backend)
++ [Importer-backend](#tidb-lightning-importer-backend)
```
Comment (TomShawn) [inline tidb-lightning/tidb-lightning-backends.md:12]: @gozssky @kennytm 所以这里没有默认的 backend 了吗？

Comment (kennytm) [reply tidb-lightning/tidb-lightning-backends.md:12]: @TomShawn yes

**Pull request merged**